src/js/service/ImportService.js

```diff
@@ -130,4 +130,68 @@
       return pskl.utils.FrameUtils.createFromImage(resizedImage);
     });
   };
+
+  /**
+   * @param {!pskl.model.Piskel} originalPiskel
+   * @param {!pskl.model.Piskel} additionalPiskel
+   * @returns {pskl.model.Piskel}
+   */
+  // Adds frames from an additionalPiskel to an originalPiskel, adjusting the size of
+  // both piskels to the largest dimensions of either piskel, and centering the content.
+  // This will only work when there is only one layer.
+  // Multi layer animations are disabled from the UI in code-dot-org/piskel
+  ns.ImportService.prototype.mergePiskels = function(originalPiskel, additionalPiskel) {
+    if (originalPiskel.layers.length != 1 || originalPiskel.layers.length != 1) {
+      throw new Error(
+        'mergePiskels was provided with piskel parameters with more than one layer');
+    }
+
+    var maxWidth = Math.max(originalPiskel.width, additionalPiskel.width);
+    var maxHeight = Math.max(originalPiskel.height, additionalPiskel.height);
+
+    if (additionalPiskel.width < maxWidth || additionalPiskel.height < maxHeight) {
+      this.resizePiskel(additionalPiskel, maxWidth, maxHeight);
+    }
+
+    if (originalPiskel.width < maxWidth || originalPiskel.height < maxHeight) {
+      this.resizePiskel(originalPiskel, maxWidth, maxHeight);
+    }
```

Comment:
Suggestion (not required): I would push the repeated `width < maxWidth || height < maxHeight` check here down into `resizePiskel` because it's slightly [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)er and it's sensible that the operation would be a no-op if the dimensions already match.  I'd also rename `resizePiskel` to something like `setPiskelSize` to parallel the helper `setFrameSize` below.